src/clean_env_before_mining.py

```diff
@@ -37,25 +31,26 @@ def clean_directory_content(script_dir: str, directory: str) -> None:
         shutil.rmtree(directory_path)
 
 
-def clean_environment(env_vars):
-    print("Cleaning of env for Living Documentation started")
+def clean_environment():
+    print("Cleaning environment for the Living Doc Generator")
 
     # Get the directory of the current script
     script_dir = os.path.dirname(os.path.abspath(__file__))
 
+    # Get the directory variables from the environment variables
+    fetch_directory = os.environ['FETCH_DIRECTORY']
+    consolidation_directory = os.environ['CONSOLIDATION_DIRECTORY']
+    markdown_page_directory = os.environ['MARKDOWN_PAGE_DIRECTORY']
+
     # Clean the fetched data directories
-    clean_directory_content(script_dir, FETCH_DIRECTORY)
-    clean_directory_content(script_dir, CONSOLIDATION_DIRECTORY)
+    clean_directory_content(script_dir, fetch_directory)
+    clean_directory_content(script_dir, consolidation_directory)
 
     # Clean the output directory
-    clean_directory_content(script_dir, MARKDOWN_PAGE_DIRECTORY)
+    clean_directory_content(script_dir, markdown_page_directory)
 
     print("Cleaning of env for Living Documentation ended")
 
 
 if __name__ == "__main__":
-    import sys
-    env_vars = sys.argv[1:]
-    clean_environment(env_vars)
-    # TODO - move dir variables definition to the controlling script - add them to the env_vars
-    #  - to able to provide them to all rest scripts #Issue: https://github.com/AbsaOSS/living-doc-generator/issues/4
+    clean_environment()
```

Comment:
```suggestion
    clean_environment()
    
```

Reply:
Fixed in commit: [7ec1572](https://github.com/AbsaOSS/living_doc_generator/commit/7ec157261c29a87d71919089c2f852834b389697)